docs/api/XDP_MATCH_TYPE.md

```diff
@@ -0,0 +1,107 @@
+
+# XDP_MATCH_TYPE structure
+
+Specifies an XDP inspection program rule match type.
+
+## Syntax
```

Comment:
This entire section looks like a mess when you view it as rendered. Wrap it in triple-backticks to preserve the formatting.